README.md

```diff
@@ -21,19 +21,24 @@ dependencies {
 ```
 
 In your `Application` class:
-
 ```java
 public class ExampleApplication extends Application {
 
+  public static RefWatcher getRefWatcher(Context context) {
+    ExampleApplication application = (ExampleApplication) context.getApplicationContext();
```

Comment:
Why not make RefWatcher a static?
This code will fail with a class cast exception if getRefWatcher() is called during backup or restore.

Reply:
@kurtisnelson it is static method. `RefWatcher` is on line 32

Reply:
then I guess we would also have to update the FAQs.
How come would it crash with a `ClassCastException`?

Reply:
If the variable were just a static, the context lookup/cast wouldn't need to happen.
The backup/restore manager will wake up your application with plain Application when running.

Reply:
ah perfect then, I will update the PR. Thanks for the tip!